examples/grafana-integration/grafana/provisioning/dashboards/dashboard.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: 1
```

Comment:
Could these files be placed under a shorter path `examples/grafana-integration/grafana/`? I think this deep nesting makes the example unnecessarily more complex. If Grafana expects a certain path inside the container, we can always map it in the docker-compose.yaml.

Reply:
I was thinking the same, although this way the services have their own folder, which makes it easier to tie the config files to their origin/purpose. 
Alternatively, I can link these from the `examples/grafana-integration/`, directly.

Reply:
Alright, let's simplify it then.